### PR TITLE
Switched out the old polyzone for ox_lib zones

### PR DESCRIPTION
**Pull request by Molicheu**
**Describe Pull request**
Switched out the old polyzone for ox_lib zones

If your PR is to fix an issue mention that issue here

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes/no] (Be honest)
- Yes
- Does your code fit the style guidelines? [yes/no]
- Yes
- Does your PR fit the contribution guidelines? [yes/no]
- Yes

### CODE REVIEW

**Review by ImXirvin — APPROVED** [review]
Looks good

**Review by YvanCywan — APPROVED** [review]

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
         with:
           capture: "junit.xml"
           args: "-t --formatter JUnit"
-          extra_libs: mysql+polyzone+qblocales
+          extra_libs: mysql+polyzone+qblocales+ox_lib
```
Comment (YvanCywan) [inline .github/workflows/lint.yml:16]: You can remove polyzone since ox_lib provides its own equivalent

**Review by YvanCywan — APPROVED** [review]

File: client/main.lua [hunk 2]
```diff
@@ -138,15 +116,16 @@ local function CreateZone(type, garage, index)
                 CreateZone("house", garage, index)
                 HouseMarkers = true
             elseif type == "house" then
-                if IsPedInAnyVehicle(PlayerPedId(), false) then
+                if IsPedInAnyVehicle(cache.ped, false) then
```
Comment (FjamZoo) [inline client/main.lua:119]: Instead of doing

`if IsPedInAnyVehicle(cache.ped, false) then`

do

`if cache.vehicle then`

**Review by FjamZoo — CHANGES_REQUESTED** [review]

**Comment by FjamZoo** [comment]
Also remove polyzone from manifest and add ox_lib as a dependency

**Review by FjamZoo — APPROVED** [review]

**Pull request merged**